hazelcast-spring/src/test/resources/com/hazelcast/spring/node-client-applicationContext-hazelcast.xml

```diff
@@ -22,7 +22,8 @@
        xsi:schemaLocation="http://www.springframework.org/schema/beans
 		http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
 		http://www.hazelcast.com/schema/spring
-		http://www.hazelcast.com/schema/spring/hazelcast-spring.xsd">
+		hazelcast-spring-3.9.xsd
```

Comment:
We specify the XSD version in the member XML as well and for me this fixes the parsing (somehow `hazelcast-spring.xsd` doesn't get resolved to 3.9 correctly).